### PR TITLE
Practice Operators and DataTypes

### PR DESCRIPTION
**Pull request by Zain-Bajwa**
# Python Programming For Beginners
1. **Getting Started**
    - [Variables](https://github.com/Zain-Bajwa/learn-python/blob/python_learning_stage1/src/getting_started/test_variables.py)
2. **Operators**
    - [Arithmetic Operators](https://github.com/Zain-Bajwa/learn-python/blob/python_learning_stage1/src/operators/test_arithmetic.py) (`+`, `-`, `*`, `/`, `//`, `%`, `**`)
    - [Bitwise Operators](https://github.com/Zain-Bajwa/learn-python/blob/python_learning_stage1/src/operators/test_bitwise.py) (`&`, `|`, `^`, `>>`, `<<`, `~`)
    - [Assignment Operators](https://github.com/Zain-Bajwa/learn-python/blob/python_learning_stage1/src/operators/test_assigment.py) (`=`, `+=`, `-=`, `/=`, `//=` etc.)
    - [Comparison Operator](https://github.com/Zain-Bajwa/learn-python/blob/python_learning_stage1/src/operators/test_comparison.py) (`==`, `!=`, `>`, `<`, `>=`, `<=`)
    - [Logical Operators](https://github.com/Zain-Bajwa/learn-python/blob/python_learning_stage1/src/operators/test_logical.py) (`and`, `or`, `not`)
    - [Identity Operators](https://github.com/Zain-Bajwa/learn-python/blob/python_learning_stage1/src/operators/test_identity.py) (`is`, `is not`)
    - [Membership Operators](https://github.com/Zain-Bajwa/learn-python/blob/python_learning_stage1/src/operators/test_membership.py) (`in`, `not in`)
3. **Data Types**
    - [Numbers](https://github.com/Zain-Bajwa/learn-python/blob/python_learning_stage1/src/data_types/test_numbers.py) (including booleans)
    - [Strings](https://github.com/Zain-Bajwa/learn-python/blob/python_learning_stage1/src/data_types/test_strings.py) and their methods
    - [Lists](https://github.com/Zain-Bajwa/learn-python/blob/python_learning_stage1/src/data_types/test_lists.py) and their methods (including list comprehensions)
    - [Tuples](https://github.com/Zain-Bajwa/learn-python/blob/python_learning_stage1/src/data_types/test_tuples.py)
    - [Sets](https://github.com/Zain-Bajwa/learn-python/blob/python_learning_stage1/src/data_types/test_sets.py) and their methods
    - [Dictionaries](https://github.com/Zain-Bajwa/learn-python/blob/python_learning_stage1/src/data_types/test_dictionaries.py)
    - [Type Casting](https://github.com/Zain-Bajwa/learn-python/blob/python_learning_stage1/src/data_types/test_type_casting.py)
### Testing Instructions
- Run the following command in your command line to install [pytest](https://docs.pytest.org/en/7.1.x/getting-started.html):
`pip install -U pytest`
- Go to the required directory and run the following command:
`pytest ./`
The above command will run all files with name `test_*.py` or `*_test.py` in your current directory.

### CODE REVIEW

**Comment by arslanashraf7** [comment]
I see that this PR is still based on the https://github.com/Zain-Bajwa/learn-python/pull/1. As per our discussions, it should have been on a new branch that doesn't include the changes from the older one.

File: src/data_types/test_dictionaries.py [hunk 1]
```diff
@@ -21,7 +21,44 @@
 """
 
 
-def test_dictionary():
+def dic_comprehension():
```
Comment (arslanashraf7) [inline src/data_types/test_dictionaries.py]: Any specific reason to change the function names. This comment applies here and on all the other places where we changed the function name.

Comment (Zain-Bajwa) [reply src/data_types/test_dictionaries.py]: It happened by mistake. Now I have changed the name of the functions. I also remove other conflicts but still, there are 6 deletions. Some of them I understand the reason. We will discuss it in standup.

**Review by arslanashraf7 — APPROVED** [review]

**Comment by arslanashraf7** [comment]
Could you please squash your commit into a single commit before you merge? Also, We could merge this without squashing but the commit messages are not clear enough so it's better to merge with squash.

**Comment by arslanashraf7** [comment]
Could you change the PR title to represent what the PR is doing? e.g. `Practice Operators and DataTypes` would be a good one.

**Pull request merged**